CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Fixed
+
+* Breakpoints and bookmarks are now updated after the buffer rewrite.
```

Comment:
I think this better qualifies as an enhancement than a fix, seeing as it wasn't a feature we were claiming to have previously.